wonjong/app.js

```diff
@@ -96,6 +96,39 @@ app.get("/user/post", async (req, res) => {
   res.status(200).json({ data: allUsersPostsViews });
 });
 
-app.listen(PORT, function () {
+app.patch('/post/patch', async (req, res) => {
+  const { postId, title, content } = req.body;
+  await appDataSource.query(
+    `UPDATE posts
+    SET title = ?,
+    content = ?
+    WHERE  posts.id = ?`,
+    [title, content, postId]
+  );
+  res.status(201).json({ message: 'postPatch' });
+});
+
+app.delete('/post/delete', async (req, res) => {
+  const { postId } = req.body;
```

Comment:
- 이미 HTTP method의 delete로 어떤 행위를 할 것인지 정해지기 때문에 endpoint에서 delete는 제거해 주셔도 될 것 같습니다.
- 아직 RESTful에 대해 배우지 않았지만, endpoint는 어떤 자원(data)에게 행위를 할지를 나타냅니다. 따라서 특정 post에 대한 정보도 URL에 나태는 것이 좋습니다. 이에 대해 더 알아보고 싶으시면 path parameter에 대해 한 번 알아보세요.

```suggestion
app.delete('/posts/:postId', async (req, res) => {
  const { postId } = req.params;
```